docs/architecture/discovery/pepr-hld.md

```diff
@@ -242,9 +242,29 @@ Note that updating an entity _may_ include changing its `status`. See a summary
 
 ### Summary Log
 
-1. `created`: summary log has been created (after upload by User) but has not yet been ingested by the service
-2. `ingested`: summary log has been ingested by the service and the prospective modifications to waste records can be reviewed by the user
-3. `approved`: summary log has been approved by the user and modifications applied to waste records
+Summary logs transition through the following states based on what has been accomplished:
+
+```mermaid
+stateDiagram-v2
+    [*] --> awaiting_upload: Upload process started
+    awaiting_upload --> uploaded: File uploaded successfully
+    awaiting_upload --> upload_failed: Virus scan failed
+    uploaded --> validating: Validation started
+    validating --> validation_failed: Validation errors found
+    validating --> validated: Validation completed successfully
+    validated --> submitted: Summary log submitted
+    upload_failed --> [*]: User must restart
+    validation_failed --> [*]: User must restart
+    submitted --> [*]
+```
+
+1. `awaiting_upload`: entity created and upload process started, awaiting file upload from user
+2. `uploaded`: file has been successfully uploaded and passed virus scan, awaiting validation
+3. `upload_failed`: file upload or virus scan has failed
+4. `validating`: validation process has started and is in progress
+5. `validation_failed`: validation has completed and found errors
+6. `validated`: validation has completed successfully, prospective modifications are ready for user review
```

Comment:
nitpick: the past tense words `uploaded` and `validated` feel out of sync with `upload_failed` and `validation_failed`